addons/flannel/template/base/yaml/kustomization.yaml

```diff
@@ -2,5 +2,5 @@ resources:
   - kube-flannel.yml
   - troubleshoot.yaml
 
-patches:
+patchesStrategicMerge:
```

Comment:
This won't work with kubernetes >= 1.27.x due to Kustomize v5 removing support for patchesStrategicMerge.
Instead you can do:
```yaml
patches:
- path: kube-flannel-cfg.patch.yaml
```

Reply:
hah ok